### ISSUE TITLE
Can't get ActionName when call request.GetActionName() in Elasticsearch

### ISSUE THREAD

**Issue by Lexsss** [aliyun/alibaba-cloud-sdk-go#289]
* **Product and API**:
`elasticsearchClient.DescribeInstance`
* **Platform**:
* **How to Reproduce**
```
request := elasticsearch.CreateDescribeInstanceRequest()
addDebug(request.GetActionName(), raw)
```

Got "" when call request.GetActionName(), Please fix it.

**Comment by wenzuochao** [comment]
> * **Product and API**:
>   `elasticsearchClient.DescribeInstance`
> * **Platform**:
> * **How to Reproduce**
> 
> ```
> request := elasticsearch.CreateDescribeInstanceRequest()
> addDebug(request.GetActionName(), raw)
> ```
> 
> Got "" when call request.GetActionName(), Please fix it.

I have fixed it in #290 ,Thanks for your support for our project.

**Comment by wenzuochao** [comment]
The pr which fixed the error have been merged, you can update the latest codes and try again. I will close the issue. Thanks.
